src/DetailView/EventTabs.js

```diff
@@ -5,12 +5,38 @@ import { css } from 'aphrodite';
 injectTapEventPlugin();
 
 class EventTabs extends Component {
+
+	componentWillMount() {
+		if (this.props.eventTypes.length > 0) {
+			const firstEventTypeId = this.props.eventTypes[0].Id;
+			this.props.eventTypeChangeHandler(firstEventTypeId);
+		}
+	}
+
+	getTabs() {
+		if (this.props.eventTypes.length > 0) {
+			return (
+				this.props.eventTypes.map((eventType, key) => {
+					return (
+						<Tab
+							label={eventType.Name}
+							onClick={() => this.props.eventTypeChangeHandler((eventType.Id))}
```

Comment:
It is possible but since ES6 it seems to be common to use an arrow function:
http://stackoverflow.com/questions/29810914/react-js-onclick-cant-pass-value-to-method